.github/workflows/ci-test.yml

```
@@ -42,7 +42,7 @@ jobs:
           poetry run coverage report && \
           poetry run coverage xml -o cov.xml
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v2
+        uses: codecov/codecov-action@v4
```

Revert breaking change to make tests pass.
```suggestion
        uses: codecov/codecov-action@v2
```